scripts/docker/fedora.24/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora:24
 
-RUN dnf install -y bash git cmake wget which python clang-3.8.1-1.fc24.x86_64 llvm-devel-3.8.1-2.fc24.x86_64 make libicu-devel lldb-devel.x86_64 \
+RUN dnf install -y bash git cmake wget which python clang-3.8.0-1.fc24.x86_64 llvm-devel-3.8.0-1.fc24.x86_64 make libicu-devel lldb-devel.x86_64 \
```

Comment:
This regresses the clang version used to ship Core-Setup repo binaries for 1.1.0. See https://github.com/naamunds/core-setup/blob/release/1.1.0/scripts/docker/fedora.24/Dockerfile#L3.